frontend/packages/kubevirt-plugin/integration-tests/tests/vm.wizard.scenario.ts

```diff
@@ -221,4 +227,25 @@ describe('Kubevirt create VM using wizard', () => {
     },
     CLONE_VM_TIMEOUT_SECS,
   );
+
+  it('ICNV-5045 - dont let the user continue If PXE provision source is selected on a cluster without a NAD available', async () => {
+    deleteResources([multusNAD]);
+    const vm = new VMBuilder(getBasicVMBuilder()).build();
+    const wizard = new Wizard();
+    await vm.navigateToListView();
+    await isLoaded();
+    await wizard.openWizard(null);
```

Comment:
you can pass the VM model or nothing instead

Reply:
I tried, it force me to send one argument

Reply:
I see, it is not optional - in that case can we send the model or make it optional?